apps/mymedicare_cb/views.py

```diff
@@ -0,0 +1,150 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from django.conf import settings
+from django.http import HttpResponseRedirect
+from django.core.urlresolvers import reverse
+from django.contrib.auth.models import User, Group
+from django.contrib.auth import login
+import requests
+from django.http import HttpResponse
+from apps.accounts.models import UserProfile
+from apps.fhir.bluebutton.models import Crosswalk
+from apps.fhir.bluebutton.utils import get_resourcerouter
+import urllib.request as req
+import random
+from .models import AnonUserState
+
+__author__ = "Alan Viars"
+
+
+def callback(request):
+    token_endpoint = getattr(
+        settings, 'SLS_TOKEN_ENDPOINT', 'https://dev.accounts.cms.gov/v1/oauth/token')
+    redirect_uri = getattr(settings, 'SLS_REDIRECT_URI',
+                           'http://localhost:8000/mymedicare/sls-callback')
+    userinfo_endpoint = getattr(
+        settings, 'SLS_USERINFO_ENDPOINT', 'https://dev.accounts.cms.gov/v1/oauth/userinfo')
```

Comment:
These should not default to lower environment values. Would be better to crash in these situations.